js/index.js

```diff
@@ -1,6 +1,95 @@
 // Iteration 1: Names and Input
 
+const hacker1 = "Zero Cool"
+
+console.log(`The drive's name is ${hacker1}`)
+
+const hacker2 = "Acid Burn"
+
+console.log(`The navigator's name is ${hacker2}`)
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log('The driver has the longest name, it has XX characters.')
+} else if (hacker1.length < hacker2.length) {
+    console.log('It seems that the navigator has the longest name, it has XX characters.')
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
```

Comment:
talvez esse `else` nem seja necessário, porque se não for nem maior nem menor, com certeza será igual. Se quiser faça o teste e me avisa se der certo 😉